src/OrchardCore.Modules/OrchardCore.Media.AmazonS3/AwsStorageOptionsConfiguration.cs

```diff
@@ -50,7 +50,8 @@ private void ParseBucketName(AwsStorageOptions options, TemplateContext template
 
             options.BucketName = template
                 .Render(templateContext, NullEncoder.Default)
-                .Replace("\r", String.Empty).Replace("\n", String.Empty);
+                .Replace(System.Environment.NewLine, String.Empty)
+                .Trim();
```

Comment:
I don't think we need to trim the string if we remove the new line, unless there' another white-space. Just leave it as its